graylog2-web-interface/src/actions/roles/AuthzRolesActions.js

```diff
@@ -6,6 +6,7 @@ import { singletonActions } from 'views/logic/singleton';
 import type { RefluxActions } from 'stores/StoreTypes';
 
 type AuthzRolesActionsType = RefluxActions<{
+  deleteRole: (roleId: string) => Promise<string[]>,
```

Comment:
The type for the response `string[]` is based on `deleteUser`. When we the api route exists, we may need to adjust this.